metron-platform/metron-integration-test/src/main/config/zookeeper/global.json

```diff
@@ -7,10 +7,12 @@
   "solr.collection": "metron",
   "solr.numShards": 1,
   "solr.replicationFactor": 1,
+  "flushIntervalInMs": "20000",
+  "flushOnTimeout": "false",
```

Comment:
Agree that flushIntervalInMs is dependent on other flag but if we think from user perspective both have different meaning.